index.html

```diff
@@ -27,7 +27,7 @@
         }
         
         .block {
-            background-color: #efefef;
+            background-color: black;
```

Comment:
Please try to avoid direct color name use # value
```suggestion
            background-color: #000000;
```